### ISSUE TITLE
Make entire suite platform-agnostic

### ISSUE THREAD

**Issue by foxcpp** [foxcpp/sutrc#8]
Things to consider:
- [ ] `sutagent-windows` is generally Windows-only
  Due to amount of Windows-specific code in it I would prefer to move more code to `agent` package and write separate implementation for POSIX platforms (`sutagent-posix`, probably).
- [ ] Process killing is bound to Windows's `taskkill` command
  => Introduce new task type
- [ ] Power control is bound to Windows's `shutdown` command
  => Introduce new task type
- [ ] sutcp assumes `\` path separator in filesystem operations
  => Probably consider implementing "universal path separator" i.e. use `/` everywhere and convert to platform-specific separator only at agent level.

Depends on #5 because we definitely want to deploy backwards-incompatible protocol changes without big problems.

CC @Hexawolf

**Comment by Hexawolf** [comment]
In Go environment it is a good idea to use `// +built <OS>` thing, it was made for this purpose. I recommend moving generally shared code to `agent` package.
